packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
the `.gen` name is weird until it is actually generated in #43692, but I think pulling out the breaking api change into its own smaller PR makes this more clear